### ISSUE TITLE
PAMAC hurts Gnome-Shell performance

### ISSUE THREAD

**Issue by leipero** [manjaro/pamac#326]
As title says, after about 23 seconds when gnome session is loaded, whole shell starts lagging, it's always after 23 seconds since "enter" is pressed at GDM login screen, I assume pamac loads something at that time (20 secs after login), the problem is, it is not just check, since performance issue does not go away. Even worse, removing pamac completely doesn't solve the problem, and after reboot issue is back.

Demonstration (download video to watch it in 60FPS so you can see the problem):
https://drive.google.com/open?id=0B_j0gO_cc4rXWDVDVkFnTmN6Mlk

**Comment by Th3Z0ne** [comment]
Using Manjaro-Gnome myself I can not reproduce the issue you are having. Are you sure it is pamac? I mean you removed it and the actual problem stays - so your sure it's pamac?

**Comment by leipero** [comment]
Yeah, I am sue it is pamac, however, I used it on Arch, not Manjaro, maybe that si the reason why you can't reproduce it (not sure how much repositories of manjaro differs)?

I would love to test it more, however, after I build ir from AUR, and even after removing it, it still kept the bug for some reason (but I didn't tested much).

One thing that may help nailing down the problem is that at one build (I've installed it twice) it actually didn't caused bug, but pamac would never report new updates, unless I run pacman -Sy, then pamac would report new updates, so auto-check did not work at the time when bug was not there while pamac was installed.

The last version I've tested is commit 3df93a4da7a0d93492a139d09504f7883aa4cef9 in AUR (pamac-aur), there are two versions after that on 3rd and 4th october, if you can reproduce with that version, it might be already fixed.

**Comment by guinux** [comment]
If the problem persists after removing pamac and a reboot, it's impossible that your problem is pamac so I close it.

**Comment by leipero** [comment]
Well maybe I was caught in some mid phase when someting was updated in systemd itself, I had to re-install system (twice) so I wasn't that keen to try it again.

I've installed it now and it works properly, why I think it was pamac (or something connected to it)? Because after 20 secs it gets loaded, at that time, if I move my mouse pointer over the shell (as in the video) constantly, i can see one small stutter at the time when it's loaded. At the time when this bug happened, pamac would check for updates and report them afterwards, but whole shell would stay stuttery, however, on 2nd install, pamac wasn't able to check updates (it wold never show them) but bug wasn't there. Now for sure, that could be something related to other components of pacman (or dbus), can't know for sure.

Anyway, long story short, now it works properly, it does that "small stutter" when it's loaded, but it is smooth and clean afterwards, and it does check updates. So this bug can stay closed, and if it re-occurs I will investigate it further and post it here.
